woocommerce/woocommerce-android
# [JITM] JITM to Orders

Closes: #8816


### Description

Adds JITM to the orders screen. It can be shown only after IPP and SimplePayments + Order creation feedback banners are closed. Also, for now, it's behind the beta feature flag

### Testing instructions

Run clean build
* Notice that IPP feedback banner is shown. (use store from a country that has IPP available) Dismiss it
* Notice that the Order Creation banner is shown. Dismiss it
* Notice that JITM is shown (make sure that beta TTP feature flag is enabled though)
The code is tricky here so please thing about other scenarios that I could miss. Use Flipper to change the settings to simulate different cases

### Images/gif


<img width="267" alt="image" src="https://user-images.githubusercontent.com/4923871/235439371-a3e7dd5b-f46a-4ff4-a601-f4cac81c3f47.png">
